Client/Frontend/Browser/TabTrayController.swift

```diff
@@ -412,22 +388,31 @@ class TabTrayController: UIViewController {
             make.height.equalTo(UIConstants.BottomToolbarHeight)
         }
     }
+    
+    func applyTheme(_ theme: Theme) {
+        collectionView?.backgroundColor = TabTrayControllerUX.BackgroundColor.colorFor(theme)
+        toolbar.applyTheme(theme)
+    }
 
 // MARK: Selectors
     @objc func didClickDone() {
-        presentingViewController!.dismiss(animated: true, completion: nil)
+        if tabDataSource.tabs.isEmpty {
+            openNewTab()
+        } else {
+            if TabType.of(tabManager.selectedTab).isPrivate != privateMode {
+                tabManager.selectTab(tabDataSource.tabs.first!)
+            }
+            self.navigationController?.popViewController(animated: true)
+        }
     }
 
     @objc func didClickAddTab() {
         openNewTab()
     }
 
     @objc func didTapLearnMore() {
-        let appVersion = Bundle.main.object(forInfoDictionaryKey: "CFBundleShortVersionString") as? String
-        if let langID = Locale.preferredLanguages.first {
-            let learnMoreRequest = URLRequest(url: "https://support.mozilla.org/1/mobile/\(appVersion ?? "0.0")/iOS/\(langID)/private-browsing-ios".asURL!)
-            openNewTab(learnMoreRequest)
-        }
+        let learnMoreRequest = URLRequest(url: URL(string: "https://github.com/brave/browser-laptop/wiki/What-a-Private-Tab-actually-does")!)
```

Comment:
You can get rid of this link, we will not be loading a remote source, we will just be adding that extra length in the tab tray. Or you just leaving it for visual aesthetics rn?

Reply:
I left this in for parity sake with 1.6, was better than having it open to the Mozilla site still 😛

I'll make a new ticket for replacing this with new copy

Reply:
Sounds good 👍